observations/forward_operators/obs_def_rttov_mod.f90

```diff
@@ -2183,7 +2189,12 @@ subroutine do_forward_model(ens_size, nlevels, flavor, location, &
          end if 
 
          if (allocated(clouds % snow)) then
-            totalice(:) = totalice(:) + max(clouds % snow(imem,lvlidx),0.0_r8)
+            ! Following Kostka et al., 2014
+            if (is_vis) then
```

Comment:
is_vis is not defined in obs_def_rttov_mod.f90 so this module does not compile: 

/glade/derecho/scratch/hkershaw/DART/pull_reqeuests/pull_896/DART/observations/forward_operators/obs_def_mod.f90(1914): error #6404: This name does not have a type, and must have an explicit type.   [IS_VIS]
            if (is_vis) then

Does this need is_vis defined the same way as rttov13:
`is_vis = sensor%sensor_type_name == 'vis'`

Reply:
This commit:  817cf37cb9357b54b3910a546d8faed8d41d669d

Reply:
It does look like the is_vis line was just missed from obs_def_mod_rttov.f90. @braczka Do you want me to go ahead and make the change to add in `is_vis = sensor%sensor_type_name == 'vis'`?

Reply:
@braczka which branch are you building, Lukas's branch (this pull request) or NCAR/radiance-docs where I fixed the compile issue?
https://github.com/NCAR/DART/compare/main...radiance-docs

Reply:
I guess it was the NCAR/radiance-docs because now that I look at the obs_def_rttov_mod.f90 it has the is_vis changes -- didn't realize those changes had been implemented given your last comment.   

Regardless - this seems to be resolved and so just go ahead as you see fit.

Reply:
The changes haven't been implemented on this pull request. Its two different branches.

Reply:
Can you give a summary of the code changes for the release notes, the description in the pull request only has the documentation rewrite.  

Thanks,
Helen